packages/templates/cra-template-desktop-viewer/template/README.md

```diff
@@ -17,7 +17,7 @@ ITWIN_VIEWER_CLIENT_ID="native-xxxxxxxx"
   - **Visualization**: `imodelaccess:read`
   - **iModels**: `imodels:read`
   - **Reality Data**: `realitydata:read`
-  - **Projects**: `projects:read`
+  - **itwins**: `itwins:read`
```

Comment:
there is no `itwins` association

Reply:
it's administration: https://developer.bentley.com/api-groups/administration/

Reply:
the `scope` is `itwins:read`
but the `association` is `Administration`

Reply:
changed